.github/workflows/build_runson_ami.yaml

```diff
@@ -0,0 +1,52 @@
+name: Build RunsOn AMI
+
+on:
+    workflow_dispatch:
+    schedule:
+      # Runs at 12:00 PM (noon) on the 1st day of every month
+      - cron: '0 12 1 * *'
+
+jobs:
+  build_ami:
+    name: Build-AMI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+        with:
+          # TODO: Rename this PAT to something that encompasses more than just cherry-picking
+          # but all PR creation (such as this one).
+          token: ${{ secrets.WORKER_PANTS_CHERRY_PICK_PAT }}
+      - name: Run packer
+        env:
+            GH_TOKEN: ${{ secrets.WORKER_PANTS_CHERRY_PICK_PAT }}
+        run: |
+          wget -O - https://apt.releases.hashicorp.com/gpg | \
+            sudo gpg --dearmor -o /usr/share/keyrings/hashicorp-archive-keyring.gpg
+          echo "deb [arch=$(dpkg --print-architecture) \
+            signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] \
+            https://apt.releases.hashicorp.com \
+            $(grep -oP '(?<=UBUNTU_CODENAME=).*' /etc/os-release || lsb_release -cs) main" | \
+            sudo tee /etc/apt/sources.list.d/hashicorp.list
+          sudo apt update && sudo apt install -y packer gh
+
+          git config --local user.email "pantsbuild+github-automation@gmail.com"
+          git config --local user.name "Worker Pants (Pantsbuild GitHub Automation Bot)"
+
+          packer init build-support/packer/runson/runson.pkr.hcl
+          packer build build-support/packer/runson/runson.pkr.hcl
+          export set AMI_ID=`jq -r '.builds[-1].artifact_id | split(":")[1]' manifest.json`
```

Comment:
Shouldn't this just be `export AMI_ID`? Why `export set`?

Reply:
Oh whoops, old habit.